onyo/commands/cat.py

```diff
@@ -21,8 +21,10 @@
 
 
 def cat(args: argparse.Namespace) -> None:
-    """
-    Print the contents of ``ASSET``\\(s) to the terminal without parsing.
+    """Print the contents of ``ASSET``\\(s) to the terminal without parsing.
```

Comment:
```suggestion
    """Print the contents of ``ASSET`` file(s) to the terminal.
```

I don't think "without parsing" adds anything useful. And the notion of an asset file seems a bit clearer to me than that of an asset, since in other context that would be the dictionary. Here it's all about the file as is.